src/pydata_sphinx_theme/theme/pydata_sphinx_theme/layout.html

```diff
@@ -63,10 +63,9 @@
           id="pst-secondary-sidebar-checkbox"/>
   <label class="overlay overlay-secondary" for="pst-secondary-sidebar-checkbox"></label>
   {# A search field pop-up that will only show when the search button is clicked #}
-  <div class="search-button__wrapper">
-    <div class="search-button__overlay"></div>
-    <div class="search-button__search-container">{% include "../components/search-field.html" %}</div>
-  </div>
+  <dialog class="search-button__search-container">
+    {% include "../components/search-field.html" %}
```

Comment:
Not for this PR but could/should this be made a macro ?

Reply:
Does Sphinx know how to handle a macro passed to `html_sidebars`?

Reply:
Ah true, if it's upstream in sphinx, likely not.